src/app/features/notification/notification-settings/notification-settings.component.ts

```diff
@@ -116,7 +116,9 @@ export class NotificationSettingsComponent implements OnInit {
   }
 
   private async saveNotificationConfig(config: NotificationConfig) {
-    await this.entityMapper.save(config);
+    await this.entityMapper.save(config).catch((err) => {
+      Logging.error(err.message);
+    });
     this.alertService.addInfo($localize`Notification settings saved.`);
```

Comment:
No!
This tells the user "settings saved" even if an error occured while saving, doesn't it?

Reply:
I have updated the code where the `Notification settings saved` alert was showing even if the data wasn't saved or if there was an error during the saving process. Now, if any error occurs while saving the data, it will display the error in the log.